nimare/tests/test_transforms.py

```diff
@@ -166,71 +166,84 @@ def test_images_to_coordinates(tmp_path, caplog, testdata_ibma, kwargs, drop_dat
     # this transformation should retain the same number of unique ids
     # unless the merge_strategy was demolish
     if img2coord.merge_strategy == "demolish":
-        expected_studies_with_coordinates = set(tst_dset.coordinates["id"]) - set(
+        expected_studies_with_coordinates = set(tst_dset.ids) - set(
             studies_without_coordinates
         )
     else:
-        expected_studies_with_coordinates = set(tst_dset.coordinates["id"])
+        expected_studies_with_coordinates = set(tst_dset.ids)
```

Comment:
These are the sources of the current test failures.

Reply:
I just need a clean way to specify which studies should have coordinates. I _might_ just hardcode the IDs.

Reply:
Should be fixed in 3e06868.